trip_app/accounts/tests/test_permissions.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from accounts import views
+from django.contrib.auth.models import AnonymousUser
+from django.urls import reverse
+
+from .user_factory import UserFactory
+
+
+@pytest.mark.django_db
+def test_user_detail_without_perm(rf):
+    """ Verify that client is unable to access GET /api/accounts/<id> if he is not admin or owner """
+    user = UserFactory()
+    url = reverse("accounts:user", kwargs={'pk': user.id})
+    request = rf.get(url)
```

Comment:
Seems like `client.get()` without a token would result in the same but in less code. Please correct me if I'm wrong

Reply:
Yep good point. Actually Django docs suggest to prefer RequestFactory over Client, and this was just a try of using RequestFactory=)  But to be consistent within tests we should use either of them two. I find Client more easy and comprehensive. Should be updated now